### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by mayanksharma3**
Adding README to repo (for DockerHub)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+## Manim DSL Compiler
+
+ManimDSL is a domain specific language using Manim to create animations for
```
Comment (rbanerjee20) [inline README.md:3]: Can we add a link to the Manim GitHub repo here?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+data structures and algorithms
```
Comment (rbanerjee20) [inline README.md:4]: Full stop

File: README.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+## Table of Contents:
+- [Installation](#installation)
+- [Usage](#usage)
+- [Documentation](#documentation)
+- [Help with Manim](#help-with-manim)
+- [Contributing](#contributing)
+
+## Installation
+
+ManimDSL produces python code that needs to run on manim to generate the mp4 files. Please  follow the
```
Comment (rbanerjee20) [inline README.md:15]: Could we capitalise Python and Manim everywhere? Would be good for consistency

File: README.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+Here is an example on how to use ManimDSL:
+
+```js
+    let stack = new Stack;
+    stack.push(1);
+    stack.pop();
+```
+
+Save this file and run the compiler on it:
```
Comment (rbanerjee20) [inline README.md:29]: Save this file as `example.manimdsl`/ Save this file with the `manimdsl` extension (and then change below code snippet accordingly to say something like: 

```
    java -jar manimdsl.jar <your-file-name>.manimdsl out.mp4
```

Comment (rbanerjee20) [inline README.md:29]: Some information on how to obtain `manimdsl.jar` would be good

File: README.md [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+
+```
+    java -jar manimdsl.jar example.manimdsl out.mp4
+```
+
+This should save the animation to out.mp4, animating the stack with 1 being pushed and popped off the stack
```
Comment (rbanerjee20) [inline README.md:35]: `out.mp4` instead of plain text

File: README.md [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+
+
+## Documentation
+Documentation is in progress at [manimdsl.github.io](https://manimdsl.github.io/).
+
+## Contributing
+Is always welcome. In particular, we need to extend the language as much as we can to produce more complex animations. For guidelines please see the
```
Comment (rbanerjee20) [inline README.md:42]: Maybe phrase as "We welcome all contributions"

Comment (rbanerjee20) [inline README.md:42]: Instead of "we need to", "we are hoping to"/ "we are looking to"

File: README.md (hunk 5 above)
Comment (rbanerjee20) [inline README.md:35]: Full stop

**Review by rbanerjee20 — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
Comment (mayanksharma3) [reply README.md:29]: Yeah I agree, we just don't know how that will be yet, including docker still too

File: README.md (hunk 1 above)
Comment (mayanksharma3) [reply README.md:3]: I've added the link to the whole repo and there's a dedicated Documentation section in the Table of Contents so it should be okay?

Comment (rbanerjee20) [reply README.md:3]: Where is the link to the whole Manim repo?

File: README.md (hunk 4 above)
Comment (mayanksharma3) [reply README.md]: I will add this in when we have a clearer idea of releases and docker image pulling

File: README.md (hunk 1 above)
Comment (mayanksharma3) [reply README.md:3]: Oh you meant the whole Manim Repo... sure I'll add that in then

**Review by rbanerjee20 — APPROVED** [review]
Looks good!

**Pull request merged**